src/components/navbar/buttons/hamburger.js

```diff
@@ -1,15 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './hamburger.scss';
+import Structure from '../../../structure.json';
+import { IoIosArrowForward } from 'react-icons/io';
+import { IoIosArrowBack } from 'react-icons/io';
+import { GrFormSearch } from 'react-icons/gr';
+
+const Hamburger = ({ showLinks }) => {
+    const [menu] = useState(Structure['menu']);
+    const [showMenu, setShowMenu] = useState(showLinks);
+
+    const [subMenu] = useState(Structure['menu'][5]);
```

Comment:
What happens if you add one more item to the menu? Will it work?

Reply:
yes, it will work, as in this case, only Features link has submenu, if we add more item to the menu, still it will not change it is index.